js/Api.elm

```diff
@@ -171,6 +172,9 @@ encodeServerAction a =
                       ]
                     )
 
+                SetName name ->
+                    ( "set_name", [ ( "name", E.string name ) ] )
```

Comment:
The server doesn't need to know about incomplete names, and the user should not be able to change the name after clicking `Ready`. Better freeze the input field when ready is pressed. Maybe let them unfreeze it by pressing ready again, but this works best if pressing Ready dents the button inward, so that the users understand that it can be unpressed.

Reply:
Done, not done.